integration-test/tests/integration_test.rs

```diff
@@ -91,3 +91,39 @@ async fn send_hello_tx() {
     let value: Value = serde_json::from_str(&query_result).unwrap();
     assert_eq!(value["data"]["account"]["seq"], 1);
 }
+
+#[actix_rt::test]
+async fn send_hello_txes() {
```

Comment:
How about making the test name to express the purpose?
This test is checking the sequence management code. Could you update the name of the test?

Reply:
@majecty Would you recommend me a nice name?

Reply:
How about `sequence_management`?
We may make tests like `sequence_management_1`, `sequence_management_2`, ...

Reply:
Ok. I'll update

Reply:
ok